Tests/KlaviyoSwiftTests/KlaviyoTestUtils.swift

```diff
@@ -156,7 +157,7 @@ extension StateChangePublisher {
     static let test = { () -> StateChangePublisher in
         StateChangePublisher.debouncedPublisher = { publisher in
             publisher
-                .debounce(for: .seconds(0), scheduler: DispatchQueue.main)
+                .debounce(for: .seconds(0), scheduler: DispatchQueue.immediate)
```

Comment:
I noticed i had used the main scheduler here. Using immediate should help speed things up.